language/move-prover/scripts/README.md

```diff
@@ -13,5 +13,5 @@ this crate, these environment variables must be set.  To set these environment v
 execute the following at the command-line after running the installation scripts described
 above:
 
-export BOOGIE_EXE=$(type -p boogie)
-export Z3_EXE=$(type -p z3)
+export BOOGIE_EXE=$(which boogie)
```

Comment:
"which boogie" does not work on my machine.  I don't understand why.  Does it work on your machine?  Because which did not work, I used "type -p".

Reply:
I updated with the actual path to the file.